docs/docs/how-tos/domain-registry.md

```diff
@@ -0,0 +1,12 @@
+---
+id: domain-registry
+title: How to set Nebari's DNS Domain Registry
+---
+
+# Nebari domain registry setup
+
+This domain will be where your application will be exposed.
+
+Currently, Nebari only supports CloudFlare for automatic DNS registration. If an alternate DNS provider is desired, change the `--dns-provider` flag from **cloudflare** to **none** on the Nebari deploy command. The deployment then will be paused when it asks for an IP address (or CNAME, if using AWS) and prompt to register the desired URL.
+
+Setting a DNS record heavily depends on the provider thus it’s not possible to have detailed docs on how to create a record on your provider. Googling setting **A/CNAME** record on your requested cloud provider should yield good results on doing it for your specific provider.
```

Comment:
```suggestion
Setting a DNS record heavily depends on the provider, and thus it’s not possible to provide detailed docs here on how to create a record on all of the providers out there. Searching "setting A DNS record" or "setting CNAME DNS record" on your specific cloud provider should yield good results.
```

I might have broken your intent with the change around search for A/CNAME record results...sorry if so!

I think the generic 'searching' is probably better than 'googling', even if the term is basically genericized at this point.